packer/rpc/server.go

```diff
@@ -140,8 +140,12 @@ func (s *Server) Serve() {
 	defer stream.Close()
 
 	h := &codec.MsgpackHandle{
-		RawToString: true,
-		WriteExt:    true,
+		BasicHandle: codec.BasicHandle{
```

Comment:
BasicHandle is deprecated, but it seems that they deprecated it 3 years ago 👀  https://github.com/ugorji/go/blame/master/codec/helper.go#L630